src/main/java/fr/esgi/al5_2/Tayarim/services/AuthService.java

```diff
@@ -3,87 +3,127 @@
 import fr.esgi.al5_2.Tayarim.auth.JwtHelper;
 import fr.esgi.al5_2.Tayarim.auth.TokenCacheService;
 import fr.esgi.al5_2.Tayarim.dto.auth.AuthLoginResponseDTO;
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.AdministrateurDTO;
 import fr.esgi.al5_2.Tayarim.dto.proprietaire.ProprietaireDTO;
-import fr.esgi.al5_2.Tayarim.entities.Proprietaire;
-import fr.esgi.al5_2.Tayarim.exceptions.*;
+import fr.esgi.al5_2.Tayarim.exceptions.AdministrateurNotFoundException;
+import fr.esgi.al5_2.Tayarim.exceptions.ProprietaireNotFoundException;
+import fr.esgi.al5_2.Tayarim.exceptions.TokenExpireOrInvalidException;
+import fr.esgi.al5_2.Tayarim.exceptions.UtilisateurNotFoundException;
 import lombok.NonNull;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.util.AbstractMap;
+import java.util.Map.Entry;
 import java.util.UUID;
 
 @Service
 @Transactional(readOnly = true)
 public class AuthService {
 
-    private final ProprietaireService proprietaireService;
-    private final JwtHelper jwtHelper;
+  private final ProprietaireService proprietaireService;
+  private final AdministrateurService administrateurService;
+  private final JwtHelper jwtHelper;
+
+  private final TokenCacheService tokenCacheService;
+
+  public AuthService(ProprietaireService proprietaireService, AdministrateurService administrateurService, JwtHelper jwtHelper, TokenCacheService tokenCacheService) {
+    this.proprietaireService = proprietaireService;
+    this.administrateurService = administrateurService;
+    this.jwtHelper = jwtHelper;
+    this.tokenCacheService = tokenCacheService;
+  }
+
+  @Transactional(readOnly = true, propagation = Propagation.NOT_SUPPORTED)
+  public AuthLoginResponseDTO login(@NonNull String email, @NonNull String password) {
+    ProprietaireDTO proprietaireDTO = null;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'proprietaireDTO' must contain no more than '1' consecutive capital letters.